osfoffline/utils/authentication.py

```diff
@@ -0,0 +1,189 @@
+import asyncio
+import datetime
+import furl
```

Comment:
Trivial FYI: furl is actually an external package, not part of the standard library